src/core/store.js

```diff
@@ -1,10 +1,23 @@
-import { applyMiddleware } from 'redux';
+import { applyMiddleware, compose } from 'redux';
 import createLogger from 'redux-logger';
 import config from 'config';
 
-export function middleware({ __config = config } = {}) {
+/*
+ * Enhance a redux store with common middleware.
+ *
+ * This returns a function that takes a single argument, `createStore`,
+ * and returns a new `createStore` function.
+ */
+export function middleware({
+  __config = config, __createLogger = createLogger,
+  __window = typeof window !== 'undefined' ? window : null,
+} = {}) {
   if (__config.get('isDevelopment')) {
-    return applyMiddleware(createLogger());
+    return compose(
+      applyMiddleware(__createLogger()),
+      __window && __window.devToolsExtension ?
+        __window.devToolsExtension() : createStore => createStore
+    );
   }
-  return undefined;
+  return createStore => createStore;
```

Comment:
We've consistently always added brackets for arrow functions since it make them more obvious. I've filed this PR to fix the missing eslint rule: https://github.com/mozilla/addons-frontend/pull/836